cli/src/config/templates.toml

```diff
@@ -39,6 +39,22 @@ log = 'builtin_log_compact'
 op_log = 'builtin_op_log_compact'
 show = 'builtin_log_detailed'
 
+op_log_node = 'if(current_operation, "@", "◉")'
+
+log_node = '''
+label("node",
+  if(!self, label("elided", "⇋"),
+    if(current_working_copy, label("wcc", "@"),
+      if(immutable, label("immutable", "◆"),
+        if(description.starts_with("wip: "), label("wip", "!"),
```

Comment:
I don't like `!` very much because it kind of looks like a continuation of the line going into it.

I haven't found anything better though